portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,9 +32,15 @@
   @Override
   public void init() {
       comments = new ArrayList<String>();
-      comments.add("You are my loop condition. I keep coming back to you.");
-      comments.add("You are my initializer: without you, my life would point to nothing (null).");
-      comments.add("You are my semicolon; always present in everything I do.");
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String newComment = request.getParameter("text-input");
+      if (!newComment.isEmpty()) {
```

Comment:
You may want to check that newComment isn't null here as well, unless you're sure that getParameter() won't return null

Reply:
It also looks like the indenting is a little off here